tests/array/first.test-d.ts

```diff
@@ -0,0 +1,54 @@
+import * as _ from 'radashi'
+import { expectTypeOf } from 'vitest'
```

Comment:
The same of https://github.com/radashi-org/radashi/pull/153#discussion_r1717949402
```suggestion
```